src/run/index.ts

```diff
@@ -7,17 +7,20 @@ import { Result } from "../rule/model";
 
 export const run = async (
   openApiPath: string,
-  requestResponsePaths: string[]
+  requestResponsePaths: string[],
+  match_server_url: string
```

Comment:
I think we can save ourselves some work by introducing an options object from the start.